app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::SessionsController < ApplicationController
+  def create
+    user = User.find_by(email: params[:email])
+
+    if user && user.authenticate(params[:password])
+      render json: { data: { token_id: user.token_id } }, status: :ok
+    else
+      render json: { error: 'Invalid email or password' }, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+      render json: { data: { message: 'Logged out successfully' } }, status: :ok
```

Comment:
Shouldn't there be some logic to find the session that is supposed to be destroyed?

Reply:
Yeah, I believe this action should be holding onto that responsibility so the session isn't just left alone when a user logs out. @mfaisalnazari can you add in that logic here? It seems like the logic for the session being created also will need to be put in the create action as well? Unless I'm missing something or misremembering how that session actually gets created.